Code/Framework/AzToolsFramework/AzToolsFramework/UI/DocumentPropertyEditor/DocumentPropertyEditorSettings.cpp

```diff
@@ -85,7 +90,7 @@ namespace AzToolsFramework
             }
         }
 
-        if (!m_expandedElementStates.empty())
+        if (m_shouldSaveSettings && !m_expandedElementStates.empty())
```

Comment:
what can currently set this to true?

Reply:
Right now this will only be true if the object was initialized with the data needed to store things in a unique registry.

It was true by default, but I'm going to swap it around so that the value is set to true in the appropriate ctor.